src/zenml/cli/integration.py

```diff
@@ -42,29 +44,10 @@ def integration() -> None:
 @integration.command(name="list", help="List the available integrations.")
 def list_integrations() -> None:
     """List all available integrations with their installation status."""
-    title("Integrations:\n")
-
-    table_rows = []
-    for name, integration_impl in integration_registry.integrations.items():
-        is_installed = integration_impl.check_installation()
-        # TODO [ENG-253]: Make the installed column right-aligned once we
-        #  add rich or some other similar dependency
-        table_rows.append(
-            {
-                "INSTALLED": "*" if is_installed else "",
-                "INTEGRATION": name,
-                "PYTHON REQUIREMENTS": integration_impl.REQUIREMENTS,
-                "SYSTEM REQUIREMENTS": list(
-                    integration_impl.SYSTEM_REQUIREMENTS.keys()
-                ),
-            }
-        )
-    print_table(table_rows)
-
-    warning(
-        "\n" + "To install the dependencies of a specific integration, type: "
+    formatted_table = format_integration_list(
+        list(integration_registry.integrations.items())
     )
-    warning("zenml integration install EXAMPLE_NAME")
```

Comment:
You think we no longer need to tell the user what he could do next?

Reply:
Good point. Not sure why I thought that was worth removing. Have added it back in now.